packages/lodestar/package.json

```diff
@@ -0,0 +1,120 @@
+{
+  "name": "@chainsafe/lodestar",
+  "version": "0.1.0",
+  "description": "A Typescript implementation of the beacon chain and validator client",
+  "author": "ChainSafe Systems",
+  "license": "GPL-3.0",
+  "bugs": {
+    "url": "https://github.com/ChainSafe/lodestar/issues"
+  },
+  "homepage": "https://github.com/ChainSafe/lodestar#readme",
+  "main": "lib/index.js",
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "build": "babel packages/beaconChain -x .ts -d lib --source-maps",
+    "build:docs": "typedoc --exclude packages/beaconChain/index.ts --out docs src",
+    "check-types": "tsc --noEmit --incremental",
+    "lint": "eslint --ext .ts packages/beaconChain/",
+    "lint-fix": "eslint --ext .ts packages/beaconChain/ --fix",
+    "pretest": "yarn run check-types",
+    "test:unit": "nyc --cache-dir .nyc_output/.cache -r lcov -e .ts mocha -r ts-node/register 'packages/beaconChain/test/unit/**/*.test.ts' && nyc report",
+    "test:e2e": "mocha --timeout 5000 -r ./.babel-register 'packages/beaconChain/test/e2e/**/*.test.ts'",
+    "test:spec": "mocha -r ./.babel-register 'packages/beaconChain/test/spec/**/*.test.ts'",
+    "test": "yarn test:unit && yarn test:spec && yarn test:e2e",
+    "coverage": "codecov"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com:ChainSafe/lodestar.git"
+  },
+  "keywords": [
+    "Ethereum",
+    "beacon",
+    "future",
+    "p2p",
+    "blockchain"
+  ],
+  "dependencies": {
```

Comment:
add @chainsafe/eth2-types dependency with version from package.json (it doesn't have to be published, lerna handles that)
Although if I remember correctly you have to add it using lerna

Reply:
So oddly emough its not working out..

Reply:
lerna add @chainsafe/eth2-type --dev --scope lodestar ?

Reply:
,lol ya just finished tis